host/bcm283x/gpio.go

```diff
@@ -479,11 +480,57 @@ func (p *Pin) PWM(duty gpio.Duty, period time.Duration) error {
 		old := pwmMemory.ctl
 		pwmMemory.ctl = (old & ^(0xff << shift)) | ((pwm1Enable | pwm1MS) << shift)
 	}
-
+	p.usingClock = true
 	p.setFunction(f)
 	return nil
 }
 
+// StreamIn implements gpiostream.PinIn.
+//
+// DMA driven StreamOut is avaiable for GPIO0 to GPIO31 pin and the resolution is
```

Comment:
available*
"the resolution is 200kHz" maybe "maximum" ?
"kHz" is the right way to write the unit. I'll do a separate commit to fix the inconsistencies in the code base. :)

Reply:
Done.